views/layouts/dashboard.handlebars

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Donatia - Dashboard</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.7.6/handlebars.min.js" integrity="sha256-usTqAE1ywvdMtksWzdeWzD75MsfJN0h0U7y2NtZL3N0=" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/static/css/general.css">
+    <link rel="stylesheet" href="/static/css/dashboard.css">
+</head>
+<body>
+ 
+    {{> navbar }}
+
+    <div id="dashboard-title">
+        {{ id }}'s Dashboard
+    </div>
+
+    <dashboard-section>
```

Comment:
I think we should just use classnames instead and use ```<div>```. Is ```dashboard-section``` a standard tag name?